### DISCUSSION TITLE
how to host it on our own server

### DISCUSSION THREAD

**Discussion by darkworks** [WebCrateApp/webcrate#59] [Ideas]
Am wondering is it possible to host the app on own server or hosting space. is the deta.space site seems not working also not reliable. 
will be good to have freedom like host on own hosting space and sync data to github account or maybe gmail drive or firebase etc

currently deta.space giving me error not able to test
`Error: Cert not issued yet, please try again after a few minutes. If the error persists, please login again and contact us.`

![image](https://user-images.githubusercontent.com/12841788/168593651-f59c6f1e-b5e1-41c0-a85a-f2e3f98ba8b7.png)

**Comment by aavshr** [comment]
hi @darkworks , an engineer from @deta here. 

Very sorry that you experienced this, can you please try logging out and logging in again and see if this resolves the issue? 

If not, can you please provide me with the `username` you used to sign up? You send me an email aavash@deta.sh if you want this information to not be public or join the webcrate discord https://webcrate.app/discord and contact me.

**Reply by darkworks** [reply]
> `username`

no luck with logout and login ,  anyway username is  : darkc99

**Reply by aavshr** [reply]
thank you and sorry again, we'll investigate further on why this happened, however you should be able to install without any issues now, can you please try once again?
